3-methyl-heptane/Wang_phi_0.5_m-h_0.0122_xO2_0.2075_NUIG.yaml

```diff
@@ -0,0 +1,147 @@
+common-properties:
+    composition: &comp
+      kind: mole fraction
+      species:
+        - species-name: 3-methyl-heptane
+          InChI: 1S/C8H18/c1-4-6-7-8(3)5-2/h8H,4-7H2,1-3H3
+          amount:
+            - 0.0122 
+        - species-name: O2
+          InChI:  1S/O2/c1-2
+          amount:
+            - 0.2075 
+        - species-name: N2
+          InChI: 1S/N2/c1-2
+          amount:
+            - 0.7803 
+    ignition-type:  &ign 
+        target: OH*
+        type: d/dt max
+datapoints:
+    - temperature:
+        - 1086.0 kelvin
+      ignition-delay:
+        - 984.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 6.66 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1089.0 kelvin
+      ignition-delay:
+        - 1072.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 7.11 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1169.0 kelvin
+      ignition-delay:
+        - 560.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 7.0 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1253.0 kelvin
+      ignition-delay:
+        - 151.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 6.56 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1261.0 kelvin
+      ignition-delay:
+        - 116.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 6.04 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1058.0 kelvin
+      ignition-delay:
+        - 1625.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 10.79 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1097.0 kelvin
+      ignition-delay:
+        - 700.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 9.47 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1211.0 kelvin
+      ignition-delay:
+        - 192.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 9.38 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1311.0 kelvin
+      ignition-delay:
+        - 54.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 8.24 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
```

Comment:
Can you make sure to have a newline at the end of each file?

Reply:
I know you said a new line, not two new lines, but it still reminds me of this wonderful RMG feature: https://github.com/ReactionMechanismGenerator/RMG-Py/issues/798
But RMG is in good company. Gaussian also requires blank lines at the end of the input file; the program will crash and dump core if they are missing. 😭

Reply:
https://xkcd.com/1172/

Reply:
But ending the file with a newline is a git best practice, nothing to do with our format (AFAIK!) 😃 See: https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file and especially the comments on the first answer